components/chip-data-view.js

```diff
@@ -15,36 +13,27 @@ const ChipDataBarChart = dynamic(() => import("./chip-data-bar-chart"), {
  * This is the view for display ChIP-seq data for a variant.
  */
 export function ChipDataView({ chipData }) {
-  const [showChipData, setShowChipData] = useState(false);
-  const router = useRouter();
-  useEffect(() => {
-    const isChip = router.asPath.endsWith(`#!chip`);
-    setShowChipData(isChip);
-  }, [router]);
```

Comment:
Since you use this sequence of state and router in more than one place, it’s a pretty good candidate for a custom hook.

Reply:
Thanks, I will keep this in mind. I can update this in my next ticket.